### ISSUE TITLE
Every evaluation causes a new janet repl to be created

### ISSUE THREAD

**Issue by livtanong** [janet-lang/vscode-janet#7]
Even if an existing janet repl exists from a previous `Janet: Evaluate expression` or `Janet: Start REPL`, a new one is created every time `Janet: Evaluate expression` is invoked.

**Comment by sogaiu** [comment]
I confirm it happens for me too with VSCodium 1.58.2 on a Linux box.

I don't remember it being this way before.  Not too sure though.

**Comment by lpedrosa** [comment]
Just opened a PR that should fix that. A property must've changed on newer API versions. 

@bakpakin let me know if there's anything else you want me to do so we can merge this fix in.

**Comment by livtanong** [comment]
Hello! This is happening again on janet 1.20 and vscode-janet fdae16. 😅

**Comment by sogaiu** [comment]
I tried on a Linux with vscodium 1.63.2 and vscode-janet fdae16 but I didn't manage to reproduce.

FWIW, the setup instructions I followed were:

* make sure vscodium is not running
* `cd ~/.vscode-oss/extensions`
* remove `~/.vscode-oss/extensions/vscode-janet`
* `git clone https://github.com/janet-lang/vscode-janet`
* start vscodium

I did find that closing any existing terminals in vscodium seemed to make things work better and the inital repl output seemed a bit weird but apart from that evaluations seemed ok (i.e. they weren't leading to multiple repl creations).
